static/app/views/sharedGroupDetails/index.tsx

```diff
@@ -13,17 +13,15 @@ import SentryDocumentTitle from 'sentry/components/sentryDocumentTitle';
 import {t} from 'sentry/locale';
 import SentryTypes from 'sentry/sentryTypes';
 import space from 'sentry/styles/space';
-import {Group, Organization} from 'sentry/types';
+import {Group} from 'sentry/types';
 import withApi from 'sentry/utils/withApi';
-import withOrganization from 'sentry/utils/withOrganization';
 
 import {OrganizationContext} from '../organizationContext';
 
 import SharedGroupHeader from './sharedGroupHeader';
 
-type Props = RouteComponentProps<{shareId: string}, {}> & {
+type Props = RouteComponentProps<{orgId: string | undefined; shareId: string}, {}> & {
```

Comment:
```suggestion
type Props = RouteComponentProps<{shareId: string; orgId?: string}, {}> & {
```
You might need to do this to appease typescript.